README.md

```diff
@@ -56,6 +57,13 @@ set :deploy_via, :jenkins_artifact
 * xz
 * raw
 
+### `release_name_from` option
+
+You can set either `'build_at'` (default) or `'deploy_at'` to this option.
```

Comment:
I think using symbol(`:build_at` or `:deploy_at`) instead of string(`'build_at'` or `'deploy_at'`) is cool.